data/lu/cfl-hafas-mgate.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Société Nationale des Chemins de Fer Luxembourgeois (CFL)",
+  "type": {
+    "hafasMgate": true
+  },
+  "supportedLanguages": [
+    "fr",
+    "en",
+    "de"
+  ],
+  "timezone": "Europe/Berlin",
+  "attribution": {
```

Comment:
Is CFL managed by DB? This looks like a copy-paste relict to me

Reply:
Fixed it, thanks.